src/Microsoft.AspNetCore.Http.Connections/HttpConnectionContext.cs

```diff
@@ -124,7 +127,7 @@ public void TickHeartbeat()
             }
         }
 
-        public async Task DisposeAsync()
+        public async Task DisposeAsync(bool closeGracefully = false, ILogger logger = null)
```

Comment:
I feel like putting the logger in the constructor is cleaner but I'm not going to argue about it.